lib/Dialect/TritonGPU/Transforms/AccelerateMatmul.cpp

```diff
@@ -184,10 +206,10 @@ class BlockedToMFMA : public mlir::RewritePattern {
 
     auto newAType = RankedTensorType::get(
         oldAType.getShape(), oldAType.getElementType(),
-        triton::gpu::DotOperandEncodingAttr::get(ctx, 0, mfmaEnc));
+        triton::gpu::DotOperandEncodingAttr::get(ctx, 0, mfmaEnc, kDim));
```

Comment:
Is this calling some default constructor so that `kWidth` is set directly from kDim?

Reply:
yes, this calls default constructor, which simply takes values for all parameters of dot op attribute.